src/storage-types/src/dyncfgs.rs

```diff
@@ -175,7 +175,7 @@ pub const STORAGE_UPSERT_PREVENT_SNAPSHOT_BUFFERING: Config<bool> = Config::new(
 /// Whether to enable the merge operator in upsert for the RocksDB backend.
 pub const STORAGE_ROCKSDB_USE_MERGE_OPERATOR: Config<bool> = Config::new(
     "storage_rocksdb_use_merge_operator",
-    false,
+    true,
```

Comment:
is this related to this PR?

Reply:
ah - not really but when I was debugging the merge operator I realized that we have this flag turned on by default in all environments (including production) but had left the in-code default to false, such that we werent exercising it in tests. I figured it made sense just to turn it on